src/main/java/org/apache/nifi/components/PropertyDescriptor.java

```diff
@@ -247,6 +247,8 @@ public Builder fromPropertyDescriptor(final PropertyDescriptor specDescriptor) {
          *
          * @param displayName of the property
          * @return the builder
+         * @deprecated only {@link #name(String)} should be used - this method will
+         *             be removed in the next major release
```

Comment:
Is there a reason you are only stating this in the javadocs and not using the `@Deprecated` annotation?

Reply:
Correct. I don't expect the actual removal of it before a long time so only using at documentation level makes it less noisy for now but still informs developers that they should not use it.